src/Storage/factories/PostModelFactory.php

```diff
@@ -12,7 +12,10 @@
         'featured_image'         => $faker->imageUrl(),
         'featured_image_caption' => $faker->sentence,
         'user_id'                => function () {
-            return factory(\Illuminate\Foundation\Auth\User::class)->create()->id;
+            $user = \App\User::inRandomOrder()->first();
+            if(is_null($user))
+                $user = factory(\App\User::class)->create();
+            return $user->id;
```

Comment:
The factory here is only utilized for the unit tests, so the line you removed should be sufficient.

Reply:
I don't understand, this factory is necessary for unit tests and therefore it should not be removed?